### ISSUE TITLE
Apparent Parsing Bug

### ISSUE THREAD

**Issue by mikelococo** [jrblevin/markdown-mode#17]
The following markdown code seems to cause a parser bug.

    ```
    {
     :user {
       :repositories [["clojars" {:sign-releases false}]]
       :plugins [
          [lein-cloverage "1.0.6"]
       ]
     }
    }
    ```

Steps to reproduce:
1. Write the above content to a file with markdown-mode disabled (or using an editor other than emacs)
2. Open the file in emacs and enable markdown-mode
3. Attempt to quit emacs via C-x C-c

Expected Behavior: Emacs quits as normal

Actual Behavior: An error on the mode-line saying...

```
Not a Tramp file name: "/"clojars"_{:sign-releases_false}.md"
```

At this point emacs starts behaving pretty strangely, and can't be closed via M-x kill-emacs. I've ended up killing the pid from the command-line and restarting.

Could this string being sending some regex into a tizzy?

**Comment by jrblevin** [comment]
I think it's mistakenly recognizing `[["clojars" {:sign-releases false}]]` as a wiki link. Recently, someone contributed a patch that searches for files in parent directories. Looks like it is searching for a file of that name in the root directory. Could you try this and let me know if you still have the error:

    (setq markdown-wiki-link-search-parent-directories nil)

The broader issue is that markdown-mode shouldn't look for wiki links inside of code blocks.

**Comment by mikelococo** [comment]
You're correct, disabling markdown-wiki-link-search-parent-directories suppresses the symptom, which makes the impact of the mischaracterizing wiki-links in code-blocks very tolerable.

**Comment by jrblevin** [comment]
I disabled this by default in 5fd759c. Better parsing of wiki links and code blocks is on the todo list.

**Comment by mikelococo** [comment]
Sounds reasonable. Thanks for your work maintaining markdown-mode.

**Comment by jrblevin** [comment]
With recent changes to improve detection of code blocks, I was able to fix this properly now in d299a37.
